recipe/meta.yaml

```diff
@@ -22,14 +22,13 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
-    - wasabi >=0.8.1,<1.1.0
+    - python
 
```

Comment:
flake8 ? 
see https://github.com/explosion/spacy-loggers/blob/v1.0.4/requirements.txt

Reply:
No, it's for linting, we do not need it